crates/transaction-emitter-lib/src/emitter/mod.rs

```diff
@@ -225,7 +225,7 @@ impl EmitJobRequest {
                     wait_millis: 0,
                     txn_expiration_time_secs: self.txn_expiration_time_secs,
                     transactions_per_account,
-                    max_submit_batch_size: 100,
+                    max_submit_batch_size: 20,
```

Comment:
this should stay at 100, this is unrelated

Reply:
IIRC, before I made this change I ran into "Mempool over capacity for account" while minting
https://github.com/aptos-labs/aptos-core/actions/runs/3139905838/jobs/5100792831

Reply:
argh, I need to fix that. but leave it at 20 for now